src/Microsoft.OData.Client/Microsoft.OData.Client.csproj

```diff
@@ -160,6 +160,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.Http" Version="8.0.0" />

```

Comment:
I'm assuming this reference is needed for `IHttpClientFactory`...

Reply:
Yes is.